data/summary/summary.go

```diff
@@ -89,10 +102,18 @@ func (c *GlucoseSummarizer[T, A]) GetMigratableUserIDs(ctx context.Context, pagi
 	return c.summaries.GetMigratableUserIDs(ctx, pagination)
 }
 
+func (c *Manager) GetRealtimePatients(ctx context.Context, userIds []string, startTime time.Time, endTime time.Time) (map[string]int, error) {
+	return c.summaries.GetRealtimePatients(ctx, userIds, startTime, endTime)
+}
+
+func (c *Manager) DeleteSummaries(ctx context.Context, userId string) error {
```

Comment:
Is this method used?

Reply:
No, but the function below it exists, i thought it a hole to not have it like all other functions.
But I don't want to expose it on the API yet today.